Containerfile.ubi9

```diff
@@ -7,7 +7,7 @@ COPY LICENSE /licenses
 
 USER 1001
 
-COPY --chown=1001:0 requirements-build.txt pyproject.toml .
+COPY --chown=1001:0 requirements-build.txt pyproject.toml ./
```

Comment:
This is because it was suggested by linter:

```
When using COPY with more than one source file, the destination must be a directory and end with a / or a \
dockerfile-utils(23)
```